src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -174,8 +174,15 @@ public MockResponse dispatch(RecordedRequest request) {
                     case "/v1/networks/38400000-8cf0-11bd-b23e-10b96e4ef00d/switches/switchId?open=true":
                     case "/v1/networks/38400000-8cf0-11bd-b23e-10b96e4ef00d/run":
                         return new MockResponse().setResponseCode(200)
-                                .addHeader("Content-Type", "application/json; charset=utf-8");
-
+                            .setBody("{\n" +
+                                "\"metrics\":{\n" +
+                                "\"network_0_iterations\":\"7\",\n" +
+                                "\"network_0_status\":\"CONVERGED\"\n" +
```

Comment:
Is it possible to add a test checking that the CONVERGED status value is correctly written in database for the study ?

Reply:
done